client/src/components/App.js

```diff
@@ -1,15 +1,25 @@
 import React, { Component } from 'react';
+import 'materialize-css/dist/css/materialize.min.css';
+import { BrowserRouter, Route } from 'react-router-dom';
+
 import SearchForm from './SearchForm';
 import Header from './Header';
+import MapContainer from './MapContainer';
+
 class App extends Component {
-  render() {
-    return (
-      <div className="container">
-        <Header />
-        <SearchForm />
-      </div>
-    );
-  }
+	render() {
```

Comment:
You've used hard tabs here (you can tell as GitHub renders them as 8 spaces). Please be consistent with tabs or spaces as you've used spaces everywhere else.

Reply:
We use prettier, which does the automatic indentation. I don't get to decide between tabs and spaces